docker/orcpub/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN lein deps
 
 ADD ./ /orcpub
 RUN printenv &&\
+	lein clean &&\
```

Comment:
As per conversation, this is not necessary as the filesystem is fresh at this point.
More in review comment.